packages/product/src/repositories/product-category.ts

```diff
@@ -394,6 +394,10 @@ export class ProductCategoryRepository extends DALUtils.MikroOrmBaseTreeReposito
       id: { $ne: targetCategoryId },
     })
 
+    const allCategories = await manager.find(ProductCategory, {
+      parent_category_id: targetParentId,
+    })
+
```

Comment:
is this added here for a reason?

Reply:
Damn, leftover from debugging. Nice catch